admin/README-SSH.md

```diff
@@ -0,0 +1,46 @@
+##Instructions on setting up your ssh configuration to allow for easy access to the test machines.##
+
+Ensure you have your public ssh key(an any other public keys) added to autorized_files and call the fix_ssh.yml ansible playbook using the command
+```
+make fix_ssh
+```
+in the admin directory.
+
+The host machine which has the test vms have the following entries.
+```
+192.168.123.5 client1
+192.168.122.200 setup
+192.168.122.100 storage0
+192.168.122.101 storage1
+```
+
+If accessing from the host machines, you can simply ssh into the machines as listed in the /etc/hosts.
```

Comment:
I am not sure whether we should include the following content regarding configurations for doing ssh via a remote host in such a README. @obnoxxx What do you think?

Reply:
That is why I put it in a separate file in the admin directory. The directory is intended to help users use the test vms manually.

Reply:
Updated patches. This set now adds ansible scripts to /devel. Will upload new PR with changes for /docs. This will create separate locations for ansible playbooks and docs.